code/modules/mining/ores_coins.dm

```diff
@@ -23,9 +23,13 @@
 	..()
 
 /obj/item/weapon/ore/Crossed(atom/movable/AM)
+	var/show_message = TRUE
+	for(var/obj/item/weapon/ore/O in loc)

```

Comment:
The best would probably to handle this on the turf, but it would end up with a really poor performance.